### PR TITLE
[AMS-538] fix image validation

### PR DESCRIPTION
**Pull request by xakep139**

### CODE REVIEW

File: src/MigrationTool/MigrationTool.csproj [hunk 1]
```diff
@@ -49,7 +49,7 @@
     <PackageReference Include="Serilog.Extensions.Logging.File" Version="1.1.0" />
     <PackageReference Include="Serilog.Settings.Configuration" Version="2.4.0" />
     <PackageReference Include="Serilog.Sinks.Literate" Version="3.0.0" />
-    <PackageReference Include="System.ValueTuple" Version="4.3.1" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0-preview2-25405-01" />
```
Comment (denisivan0v) [inline src/MigrationTool/MigrationTool.csproj]: Do we really need this?

Comment (xakep139) [inline src/MigrationTool/MigrationTool.csproj]: Not really, but there was package downgrade due to `preview1` dependency from `ImageSharp`

Comment (denisivan0v) [inline src/MigrationTool/MigrationTool.csproj]: Reference to release version of the package will give us exactly this version for our code. But ImageSharp can still use it's own referenced version. Isn't it?

Comment (xakep139) [inline src/MigrationTool/MigrationTool.csproj]: Done in https://github.com/2gis/nuclear-vstore/pull/86/commits/e02ede0320b634e71528ae3da80db661f01263b8

File: src/VStore/Sessions/SessionManagementService.cs [hunk 2]
```diff
@@ -40,14 +40,14 @@ public sealed class SessionManagementService
     {
         private const string PdfHeader = "%PDF-";
 
-        private static readonly Dictionary<FileFormat, IImageFormat> BitmapImageFormatsMap =
-            new Dictionary<FileFormat, IImageFormat>
+        private static readonly Dictionary<FileFormat, string> BitmapImageFormatsMap =
+            new Dictionary<FileFormat, string>
                 {
-                        { FileFormat.Bmp, new BmpFormat() },
```
Comment (xakep139) [inline src/VStore/Sessions/SessionManagementService.cs]: Formats are now internal types

**Pull request merged**